web-ui/src/pages/ReceivedRequestsPage.jsx

```diff
@@ -61,16 +63,6 @@ const Root = styled('div')({
     color: 'gray',
     marginTop: '4em',
     textAlign: 'center'
-  },
-  [`& .${classes.expandClose}`]: {
-    transform: 'rotate(0deg)',
-    marginLeft: 'auto',
-    transition: 'transform 0.1s linear'
-  },
-  [`& .${classes.expandOpen}`]: {
-    transform: 'rotate(180deg)',
-    transition: 'transform 0.1s linear',
-    marginLeft: 'auto'
```

Comment:
This is styled component logic which was being used to replicate the animated behavior of the icon.